.tekton/ct-server-push.yaml

```diff
@@ -8,6 +8,7 @@ metadata:
     pipelinesascode.tekton.dev/max-keep-runs: "3"
     pipelinesascode.tekton.dev/on-event: '[push]'
     pipelinesascode.tekton.dev/on-target-branch: '[redhat-v0.6.5]'
+    pipelinesascode.tekton.dev/on-cel-expression: event == "push" && target_branch == "redhat-v0.6.5" && ( ".tekton/ct-server-push.yaml".pathChanged() || "/Dockerfile.ct-server".pathChanged() )
```

Comment:
I want to call brief attention to this line for any one reviewing.
There is no set of files present on this repo for ct_server.
So until rhtap implements triggering builds manually the only way to trigger a build for this would be to change either the dockerfile or tekton file.